### ISSUE TITLE
[jorg] Update accessibility statement on the main website

### ISSUE THREAD

**Issue by pbwebdev** [joomla/joomla-websites#6]
Look at and review the accessibility statement on the main joomla.org website
http://www.joomla.org/accessibility-statement.html

This has not been looked at or worked on since its inception.

**Comment by MATsxm** [comment]
@pbwebdev and others
Does the A11yTeam is still alive?
Does the group can help on the topic?
Does the Marketing Team (or others) should communicate about it to move forward on this critical topic?
Thanks

**Comment by Sandra97** [comment]
@pbwebdev, would you accept to help me and see what we can do to improve j.org accessibility?

**Comment by mbabker** [comment]
Anyone?

**Comment by jeckodevelopment** [comment]
AFAIK, the Accessibility Team is now re-starting its work! Let's see what is going to happen

**Comment by MATsxm** [comment]
any news about a Team?
we really need an active one.

**Comment by jeckodevelopment** [comment]
thank you for the reminder. I'm going to ping them.

**Comment by mbabker** [comment]
Any particular reason it takes 13+ months to update one page?

**Comment by Sandra97** [comment]
Seems we don't have any accessibility team to help updating this page. And I'm not able to update it by myself.

**Comment by jeckodevelopment** [comment]
I'll ask about this during next PLT meeting (this week) since Accessibility Team is under PLT responsibility.

**Comment by MATsxm** [comment]
What ot having an official call for the a11y.
I mean, this is a REAL need for Joomla! and I guess/hope, we can find enthusiastic and passionate volunteers about this topic?

**Comment by jeckodevelopment** [comment]
Hopefully yes. 
Let me discuss with PLT and then we can see, if needed we can open a call for volunteers to improve what we have now.
Thank you for reporting :)

**Comment by Sandra97** [comment]
Please, is there anyone who can help updating this page? I'm not able to do it by myself.

**Comment by crystalenka** [comment]
I think the problem with updating the statement is that joomla itself and the .org site would have to comply with some sort of accessibility standards like [WCAG 2.0](https://www.w3.org/WAI/WCAG20/quickref/) - and as it is we don't. I'm trying to kick up the accessibility team again and see what I can do.

**Comment by Sandra97** [comment]
Thanks @crystalenka!

**Comment by Sandra97** [comment]
For information, Armen is working on it. So we should be able to update this embarrassing page soon.

**Comment by Sandra97** [comment]
As the page has been updated, I'm closing this issue.
Re-open if needed.
